src/front/client/index.html

```diff
@@ -794,7 +794,7 @@ <h2 class="starter-modal__title">
       window.DEFAULT_FIAT = 'USD';
       window.DefaultCountryCode = '+1';
       window.widgetName = 'own widget name';
-      window.buyViaCreditCardLink = 'https://buy.moonpay.com/'; // need replace at 'https://buy.itez.com/swaponline' by default
+      window.buyViaCreditCardLink = 'https://buy.itez.com/'; // need replace at 'https://buy.itez.com/swaponline' by default
```

Comment:
remove comment

Reply:
for what? we still need to replace with **...itez.com/swaponline** =)

Reply:
itez can ban our link again)